designsystem/src/commonMain/kotlin/org/hisp/dhis/mobile/ui/designsystem/component/InputUser.kt

```diff
@@ -0,0 +1,69 @@
+package org.hisp.dhis.mobile.ui.designsystem.component
+
+import androidx.compose.foundation.text.KeyboardOptions
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.outlined.Person
+import androidx.compose.material3.Icon
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.text.input.ImeAction
+import androidx.compose.ui.text.input.KeyboardType
+import androidx.compose.ui.text.input.TextFieldValue
+
+@Composable
+fun InputUser(
```

Comment:
What do you think about creating a test asserting data is filled and deleted properly?
And what about a screenshot testing to preserve the design?

Reply:
of course! good idea!